components/raftstore/src/store/config.rs

```diff
@@ -138,8 +138,6 @@ pub struct Config {
     #[config(hidden)]
     pub right_derive_when_split: bool,
 
-    pub allow_remove_leader: bool,
```

Comment:
Should also remove `allow-remove-leader`

Reply:
Do you mean also search for `allow-remove-leader`? I did, and it's a hidden configuration, so nothing shows up.

Reply:
It is in some test config file like `components/test_raftstore/src/common-test.toml`